lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ConfigurationManager.cs

```diff
@@ -55,6 +59,24 @@ internal sealed class ConfigurationManager
         {
             try
             {
+                _updateTaskStateConfiguration = configurationSerializer.FromString(updateTaskStateConfig);
+            }
+            catch (Exception e)
+            {
+                Exceptions.Throw(e, "Unable to deserialize update task state configuration", Logger);
```

Comment:
UpdateResult is already accessible form IMRU Example. User task function uses it to hold the result. The task needs to access those property so that to know what kind of result it contains at the moment in task state management. 
I will add the document.